tests/appsec/app.py

```diff
@@ -242,6 +242,90 @@ def iast_ast_patching_non_re():
     return resp
 
 
+@app.route("/iast-ast-patching-re-subn", methods=["GET"])
+def iast_ast_patching_re_subn():
+    filename = request.args.get("filename")
+    style = request.args.get("style")
+    if style == "re_module":
+        changed, number = re.subn(r"_", " ", filename)
+    elif style == "re_object":
+        pattern = re.compile(r"_")
+        changed, number = pattern.subn(" ", filename)
+    resp = Response("Fail")
+    try:
+        from ddtrace.appsec._iast._taint_tracking import is_pyobject_tainted
+
+        if is_pyobject_tainted(changed):
+            resp = Response("OK")
+    except Exception as e:
+        print(e)
+    return resp
+
+
+@app.route("/iast-ast-patching-non-re-subn", methods=["GET"])
+def iast_ast_patching_non_re_subn():
```

Comment:
ast_ast_patching_re_subn and iast_ast_patching_non_re_subn are the same function, are they?

why not?
```python
@app.route("/iast-ast-patching-non-re-subn", methods=["GET"])
@app.route("/iast-ast-patching-re-subn", methods=["GET"])
def iast_ast_patching_re_subn():
   ...
```

Reply:
ditto